### PR TITLE
Issue #1154 - displaying absolute date if week > 1

### PR DESCRIPTION
**Pull request by magsout**
r ? @karlcow  for logic
r? @miketaylr for js code

### CODE REVIEW

**Comment by miketaylr** [comment]
LGTM from the JS side of things. 👍

**Comment by karlcow** [comment]
❤️ thanks @magsout

File: webcompat/static/js/lib/models/comment.js [hunk 1]
```diff
@@ -22,7 +22,9 @@ issues.Comment = Backbone.Model.extend({
       body: md.render(response.body),
       commenter: response.user.login,
       commentLinkId: 'issuecomment-' + response.id,
-      createdAt: moment(response.created_at).fromNow(),
+      createdAt:  moment(Date.now()).diff(response.created_at, 'weeks') > 1
+                  ? moment(response.created_at).format("YYYY-MM-DD")
```
Comment (karlcow) [inline webcompat/static/js/lib/models/comment.js]: single quote comment

Comment (miketaylr) [inline webcompat/static/js/lib/models/comment.js]: nice catch. :)

Comment (magsout) [inline webcompat/static/js/lib/models/comment.js]: Ah ah yes.

**Comment by miketaylr** [comment]
> fix the quote and we can ride on and merge. :)

@magsout I can push a fixup commit for you if you're swamped, just let me know.

**Comment by magsout** [comment]
done @miketaylr 😢

**Comment by miketaylr** [comment]
Awesome!

**Comment by miketaylr** [comment]
Last push is green, let's mererrrrrge.

**Pull request merged**